ci/provisioning/post_provision_config_nodes_LEAP_15.sh

```diff
@@ -4,6 +4,7 @@ REPOS_DIR=/etc/dnf.repos.d
 DISTRO_NAME=leap15
 LSB_RELEASE=lsb-release
 EXCLUDE_UPGRADE=fuse,fuse-libs,fuse-devel,mercury,daos,daos-\*
+read -r -a inst_rpms <<< "$INST_RPMS"
```

Comment:
(lint) Possible misspelling: INST_RPMS may not be assigned, but inst_rpms is. [SC2153]